e2e/ls_test.go

```diff
@@ -786,3 +786,20 @@ func TestListNestedLocalFolders(t *testing.T) {
 		2: match(filepath.ToSlash("file.txt")),
 	}, trimMatch(dateRe), alignment(true))
 }
+
+// ls empty bucket
+func TestEmptyBucket(t *testing.T) {
+	t.Parallel()
+
+	s3client, s5cmd := setup(t)
+
+	bucket := s3BucketFromTestName(t)
+	createBucket(t, s3client, bucket)
+
+	cmd := s5cmd("ls", "s3://"+bucket)
+	result := icmd.RunCmd(cmd)
+
+	result.Assert(t, icmd.Success)
+
+	assertLines(t, result.Stdout(), map[int]compareFunc{}, alignment(true))
```

Comment:
```suggestion
	assertLines(t, result.Stdout(), nil)
```
Since we don't expect any output, the alignment option is not needed, we can also pass a `nil` map rather than an empty one. This one is a bit of nitpick :)